org.eclipse.sisu.inject/build.target

```diff
@@ -17,19 +17,19 @@
 		<location type="Maven" includeSource="true">
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-api</artifactId>
-			<version>1.6.4</version>
```

Comment:
If those dependency changes are not strictly necessary for Java 17 let us remove those. They are potentially backwards-incompatible.